### DISCUSSION TITLE
How to determine if graph is organelle-sufficient?

### DISCUSSION THREAD

**Discussion by Erythroxylum** [Kinggerm/GetOrganelle#259] [Q&A]
Hello GO community,
Excellent program here. I am curious how we can determine if assemblies have generated organelle-sufficent graphs, as this diagnosis dictates troubleshooting actions.
I have seen references to the get_org.log.txt outputting information about this (perhaps in an older version), as well as Dr. Jin referring to the connections and node coverage in assembly graphs (Discussions 161, 163, 169). 
My current idea is that the graph should be of sufficient length (150-170kbp) with all nodes having similar coverage? Anyway, some clear points on this would be most helpful. Thanks!

**Comment by Kinggerm** [comment]
It is a very interesting and important question. However, it will be difficult to define all criteria for all datasets before analysis.
For single-chromosome circular genomes, one criterion that one may think of is a single-component connected graph without dead ends; another information source is the target graph should contain most of the target genes, with gene loss and HGT as the exception. For each organelle type, e.g., plastome, as you mentioned, there is an expected size that can also be helpful for the verification of the recognized graph.
The nodes will generally follow a similar coverage, however, there are also many exceptions. For example, nodes of repeats with high multiplicity can have high coverage. Besides, some samples may have a heterogeneous distribution of reads.
